webpack/webpack.config.base.js

```diff
@@ -37,7 +37,7 @@ module.exports = validate({
 
   plugins: [
     new webpack.DefinePlugin({
-      CARDANO_API: process.env.CARDANO_API || 1,
+      CARDANO_API: process.env.CARDANO_API || 0,
```

Comment:
We have to revert this. `CARDANO_API=false` is not working since it has falsy value.